src/components/ConcurrencyLimitV2Toggle.vue

```diff
@@ -42,7 +42,7 @@
     loading.value = true
 
     try {
-      await api.concurrencyV2Limits.updateConcurrencyV2Limit(props.limit.id, { active: !value })
+      await api.concurrencyV2Limits.updateConcurrencyV2Limit(props.limit.id, { active: value })
```

Comment:
This part doesn't make sense to me. Pressing the toggle should trigger the set with the correct value shouldn't it?